src/tests/unit/events.rs

```diff
@@ -61,7 +61,7 @@ pub async fn test_get_event_ok() {
 }
 
 #[actix_rt::test]
-pub async fn test_get_event_empty() {
+pub async fn when_get_not_existing_event_expect_error() {

```

Comment:
Maybe something along the lines of `when_get_not_existing_event_expect_not_found`?